Source/WebCore/rendering/svg/RenderSVGContainer.cpp

```diff
@@ -153,18 +154,26 @@ bool RenderSVGContainer::nodeAtPoint(const HitTestRequest& request, HitTestResul
     if (!locationInContainer.intersects(visualOverflowRect))
         return false;
 
+    static NeverDestroyed<SVGVisitedRendererTracking::VisitedSet> s_visitedSet;
```

Comment:
While this new way of writing is a bit more explicit, and requires the static s_visitedSet definition on the call-site, its way easier to reason about what is happening.